src/Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -739,7 +738,9 @@ public IReadOnlyList<IPData> GetLoopbacks()
     /// <inheritdoc/>
     public IReadOnlyList<IPData> GetAllBindInterfaces(bool individualInterfaces = false)
     {
-        if (_interfaces.Count > 0 || individualInterfaces)
+        var config = _configurationManager.GetNetworkConfiguration();
+        var localNetworkAddresses = config.LocalNetworkAddresses;
+        if ((localNetworkAddresses.Length > 0 && !string.IsNullOrWhiteSpace(localNetworkAddresses[0]) && _interfaces.Count > 0) || individualInterfaces)
```

Comment:
`!string.IsNullOrWhiteSpace(localNetworkAddresses[0])`

What is this check for?

Reply:
Looks like a test for whether the list contains items, and if so, if the first item isn't just an empty string or null.
That is, testing whether the config option is set or not.

Reply:
I think a better test would be to see if any address is set, rather than just the first one?

Reply:
I'd have to find where the parsing is done to be able to tell whether that would make any difference.
Ideally it should be enough to check that the list is empty, but if the parser is naive, this kind of check is indeed needed.

Reply:
I don't remember if this check came from Em by (it's copied over from the interface init) or if I had issues in my local environment back then which required it.

I do agree though, ideally we wouldn't need it

Reply:
hey, thanks for this fix! Is there an ETA when it'll be available on the stable release?